flagd/pkg/service/flag-evaluation/connect_service.go

```diff
@@ -77,7 +77,7 @@ func NewConnectService(
 		logger:  logger,
 		eval:    evaluator,
 		metrics: mRecorder,
-		eventingConfiguration: &eventingConfiguration{
+		eventingConfiguration: &EventingConfiguration{
 			subs: make(map[interface{}]chan service.Notification),
 			mu:   &sync.RWMutex{},
 		},
```

Comment:
Moving this to NewEventingConfiguration introduces some nasty churn on this file moving to
`iservice "github.com/open-feature/flagd/core/pkg/service"`. Could make the change if desired.